spiffworkflow-frontend/src/components/ProcessInterstitial.tsx

```diff
@@ -15,12 +15,14 @@ type OwnProps = {
   processInstanceId: number;
   processInstanceShowPageUrl: string;
   allowRedirect: boolean;
```

Comment:
why not just use allowRedirect?

Reply:
Oh we could use that instead. I thought that was for redirecting to process instance page but i guess it could be used for both.